src/components/listview/listview.js

```diff
@@ -81,23 +81,26 @@ define(['itemHelper', 'mediaInfo', 'indicators', 'connectionManager', 'layoutMan
             options.tag = item.ImageTags.Primary;
             itemId = item.Id;
         }
-
-        if (item.AlbumId && item.AlbumPrimaryImageTag) {
-            options.tag = item.AlbumPrimaryImageTag;
-            itemId = item.AlbumId;
-        } else if (item.SeriesId && item.SeriesPrimaryImageTag) {
-            options.tag = item.SeriesPrimaryImageTag;
-            itemId = item.SeriesId;
-        } else if (item.ParentPrimaryImageTag) {
-            options.tag = item.ParentPrimaryImageTag;
-            itemId = item.ParentPrimaryImageItemId;
+        if (!itemId) {
+            if (item.AlbumId && item.AlbumPrimaryImageTag) {
```

Comment:
How about just `else if (item.AlbumId && item.AlbumPrimaryImageTag) {`?

Reply:
@dmitrylyzo I don't understand how I didn't thought about this before 😅. Addressed.